### PR TITLE
Chore[New Exp]: Charts for HTTP Chaos Reset Peer

### PR DESCRIPTION
**Pull request by avaakash**
Signed-off-by: avaakash [as86414@gmail.com](mailto:as86414@gmail.com)

This PR adds the charts for the HTTP Chaos Reset Peer experiment.

#### Experiment PR
- https://github.com/litmuschaos/litmus-go/pull/534

#### Docs PR
- https://github.com/litmuschaos/litmus/pull/3671

### Special notes for the reviewer
This PR also fixes some minor comments issues in HTTP latency docs

### CODE REVIEW

File: charts/generic/pod-http-reset-peer/engine.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              value: 'docker'
+
+            - name: TARGET_PODS
+              value: ''
+
+            # provide the name of container runtime
+            # for litmus LIB, it supports docker, containerd, crio
+            # for pumba LIB, it supports docker only
+            - name: CONTAINER_RUNTIME
+              value: 'docker'
```
Comment (neelanjan00) [inline charts/generic/pod-http-reset-peer/engine.yaml:39]: This ENV is repetitive, can we remove it?

File: charts/generic/pod-http-reset-peer/engine.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  name: nginx-chaos
+spec:
+  appinfo:
+    appns: 'default'
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  # It can be active/stop
+  engineState: 'active'
+  #ex. values: ns1:name=percona,ns2:run=nginx
+  auxiliaryAppInfo: ''
```
Comment (neelanjan00) [inline charts/generic/pod-http-reset-peer/engine.yaml]: Can we remove this, to maintain uniformity?

File: charts/generic/pod-http-reset-peer/engine.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  chaosServiceAccount: pod-http-reset-peer-sa
+  # It can be delete/retain
+  jobCleanUpPolicy: 'retain'
```
Comment (neelanjan00) [inline charts/generic/pod-http-reset-peer/engine.yaml]: Can we remove this, to maintain uniformity?

File: charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      email: akash.shrivastava@harness.io 
+  minKubeVersion: 1.12.0 
+  provider: 
+    name: Harness
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: latest
+  links: 
+    - name: Documentation 
+      url: https://docs.litmuschaos.io/docs/getstarted/
```
Comment (neelanjan00) [inline charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml]: Can we update the source code and exp docs here?

Comment (neelanjan00) [inline charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml]: This is also a common task across many other experiment, I will create an issue for it.

Comment (avaakash) [reply charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml]: I think all the experiments have the Documentation and Source Code fields.

File: charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  provider: 
+    name: Harness
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: latest
+  links: 
+    - name: Source Code
+      url: https://github.com/litmuschaos/litmus-go/tree/master/experiments/generic/pod-network-partition
+    - name: Documentation
+      url: https://litmuschaos.github.io/litmus/experiments/categories/pods/pod-network-partition/
```
Comment (Jonsy13) [inline charts/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml]: links may have to updated.

File: experiments/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  provider: 
+    name: Harness
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: latest
+  links: 
+    - name: Source Code
+      url: https://github.com/litmuschaos/litmus-go/tree/master/experiments/generic/pod-network-partition
+    - name: Documentation
+      url: https://litmuschaos.github.io/litmus/experiments/categories/pods/pod-network-partition/
```
Comment (Jonsy13) [inline experiments/generic/pod-http-reset-peer/pod-http-reset-peer.chartserviceversion.yaml]: Same here

**Review by Jonsy13 — CHANGES_REQUESTED** [review]

**Review by Jonsy13 — DISMISSED** [review]

**Review by uditgaurav — DISMISSED** [review]

**Review by neelanjan00 — DISMISSED** [review]

File: charts/generic/pod-http-reset-peer/rbac.yaml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+kind: ServiceAccount
+metadata:
+  name: pod-http-reset-peer-sa
+  namespace: default
+  labels:
+    name: pod-http-reset-peer-sa
+    app.kubernetes.io/part-of: litmus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```
Comment (ispeakc0de) [inline charts/generic/pod-http-reset-peer/rbac.yaml:11]: Can we add minimal permissions here?

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**